### ISSUE TITLE
"am2303_rpi" no longer available in pip repository

### ISSUE THREAD

**Issue by oxivanisher** [bk1285/rpi_wordclock#36]
It seems as the " am2303_rpi" is no longer available in the pip repository. I did not find another way to get the temperature and humidity sensor working.

**Comment by bk1285** [comment]
How about this: https://learn.adafruit.com/dht-humidity-sensing-on-raspberry-pi-with-gdocs-logging/software-install-updated

Would be great to have short feedback, if and how it works...

Regards,
Bernd

**Comment by oxivanisher** [comment]
I did all that already, but the rpi_wordclock still prints:
```
Imported plugin 5: "restart".
  Assumes no temperature sensor to be attached.
```
But, the tests from the Adafruit site are working.
```
root@wordclock:/home/pi/Adafruit_Python_DHT/examples# ./AdafruitDHT.py 2302 4
Temp=20.0*  Humidity=42.4%
```

**Comment by bk1285** [comment]
Can you check with am2302 (instead of am2303)? To me, the docs seem to be a bit inconsistent here? http://rpi-wordclock.readthedocs.io/en/latest/doc_software_setup.html#optional-dependencies-to-readout-temperature-sensor

**Comment by oxivanisher** [comment]
I was very busy the last weeks, but I will have a look at this the next week :+1:

**Comment by bk1285** [comment]
Hey @oxivanisher 

Found the dependency to be wrong...
Fix is here: f4bbcfebc22b9185c9de7d1502e8837ff00c5580

Please reopen, if the issue persists.
